### PR TITLE
Targets update

### PR DESCRIPTION
**Pull request by pawee**

### CODE REVIEW

File: src/constants/characterSettings.js [hunk 1]
```diff
@@ -1304,7 +1327,7 @@ const characterSettings = {
     [new OptimizationPlan('PvP', 5, 0, 100, 50, 0, 0, 25, 0, 25, 0, 0, 0, 0, true)]
   ),
   'URORRURRR': new CharacterSettings(
-    [optimizationStrategy["Speed, Crit, Physical Damage, Potency"].rename('PvE')]
```
Comment (grandivory) [inline src/constants/characterSettings.js:1307]: Is there no longer a PvE usage for this character? (Heck, does PvE still exist in this game?)

File: src/constants/characterSettings.js [hunk 2]
```diff
@@ -1240,10 +1256,7 @@ const characterSettings = {
     ['Troopers']
   ),
   'STORMTROOPERHAN': new CharacterSettings(
-    [
-      optimizationStrategy.Speed.rename('Speed'),
-      new OptimizationPlan('PvE', 50, 50, 50, 0, 0, 50, 0, 0, 0, 0, 0, 0, 0, true)
```
Comment (grandivory) [inline src/constants/characterSettings.js:1245]: Same here - should these targets be removed, or just deprioritized?

File: src/constants/characterSettings.js [hunk 3]
```diff
@@ -396,8 +396,8 @@ const characterSettings = {
   ),
   'DARTHSION': new CharacterSettings(
     [
-      new OptimizationPlan('PvP', 15, 15, 100, 20, 15, 0, 20, 0, 20, 0, 0, 0, 0, true),
-      new OptimizationPlan('Tanky', 10, 20, 100, 0, 15, 0, 0, 0, 0, 10, 0, 0, 100, true)
+      new OptimizationPlan('PvP', 25, 30, 100, 0, 5, 0, 0, 0, 0, 20, 0, 0, 0, true),
+      new OptimizationPlan('PvP/CA', 25, 30, 100, 0, 5, 0, 0, 0, 0, 20, 0, 0, 100, true)
```
Comment (NducTiOnomBi) [inline src/constants/characterSettings.js:412]: Would it maybe be more beneficial to just define the CA arrow primary?

File: src/constants/characterSettings.js (hunk 1 above)
Comment (pawee) [reply src/constants/characterSettings.js:1307]: Actually Tuskens are gonna be the important faction in the upcoming new Raid. But the question is it still a thing to have separate PvE PvP targets unless it's for some theorycrafted strats. (URoR is supporter, why did he has attacker target in the first place tho?)

File: src/constants/characterSettings.js (hunk 2 above)
Comment (pawee) [reply src/constants/characterSettings.js:1245]: He has his home with Sana Starros now, he gains 15% TM anytime the others are attacked.

File: src/constants/characterSettings.js (hunk 3 above)
Comment (pawee) [reply src/constants/characterSettings.js:412]: Just in case the user doesn't have  CA mods. We'd have a smoother fallback.

**Pull request merged**